protobuf/concept.proto

```diff
@@ -22,212 +22,101 @@ option java_outer_classname = "ConceptProto";
 
 package grakn.protocol;
 
-message ThingMethod {
+message ConceptManager {
+
     message Req {
-        bytes iid = 1;
         oneof req {
-            // Thing method requests
-            Thing.Delete.Req thing_delete_req = 100;
-            Thing.GetType.Req thing_getType_req = 101;
-            Thing.IsInferred.Req thing_isInferred_req = 102;
-            Thing.SetHas.Req thing_setHas_req = 103;
-            Thing.UnsetHas.Req thing_unsetHas_req = 104;
-
-            // Relation method requests
-            Relation.AddPlayer.Req relation_addPlayer_req = 200;
-            Relation.RemovePlayer.Req relation_removePlayer_req = 201;
+            GetType.Req get_type_req = 1;
+            GetThing.Req get_thing_req = 2;
+            GetRule.Req get_rule_req = 3;
+            PutEntityType.Req put_entity_type_req = 4;
+            PutAttributeType.Req put_attribute_type_req = 5;
+            PutRelationType.Req put_relation_type_req = 6;
+            PutRule.Req put_rule_req = 7;
```

Comment:
These `ConceptManager` methods from Core were originally in `Transaction` in protobuf. They are now instead grouped into a matching `ConceptManager.Req` message.